src/clf/compoundV3/compoundV3Computer.js

```diff
@@ -0,0 +1,305 @@
+const { ethers } = require('ethers');
+const dotenv = require('dotenv');
+const path = require('path');
+const { fnName, retry, getDay, roundTo, sleep } = require('../../utils/utils');
+const fs = require('fs');
+const { default: axios } = require('axios');
+dotenv.config();
+const { getBlocknumberForTimestamp } = require('../../utils/web3.utils');
+const { computeUniv3ParkinsonVolatility, getAverageLiquidityForBlockInterval } = require('../../uniswap.v3/uniswap.v3.utils');
+const { computeAggregatedVolumeFromPivot } = require('../../utils/aggregator');
+const { normalize, getConfTokenBySymbol } = require('../../utils/token.utils');
+const { compoundV3Pools, cometABI } = require('./compoundV3Computer.config');
+const { RecordMonitoring } = require('../../utils/monitoring');
+const { tokens } = require('../../global.config');
+const DATA_DIR = process.cwd() + '/data';
+const spans = [7, 30, 180];
+
+async function compoundV3Computer(fetchEveryMinutes) {
+    const start = Date.now();
+    try {
+        await RecordMonitoring({
+            'name': 'CompoundV3 CLF Computer',
```

Comment:
put the name as constant above live 

const PROCESS_NAME = 'CompoundV3 CLF Computer' and use the constant for every RecordMonitoring calls